hack/remove-calico-policy/README.md

```diff
@@ -1,4 +1,41 @@
-# Disabling and Removing Calico Policy
+# Disabling Calico Policy
+
+This guide explains two methods for disabling Calico policy on a running cluster. These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+- [Overriding Calico Policy] describes how to add high-priority Calico System Network Policy which
+  overrides the configured Network Policy.
+- [Disabling and Removing Calico Policy] describes how to fully disable Calico policy (removing all 
+  Calico-programmed policy from each node.
+
+## Overriding Calico Policy
```

Comment:
If this is a separate MD then we either have the option of:
-  creating a whole new directory specifically for this option
-  add another MD which won't be called README.md (but keep everything in the same dir)
-  add a separate README.md doc and move the existing README.md to a new MD doc (so the README is the landing page and everything is in the same dir).

For general docs flow I would go with the latter.  Agree?